cfgov/v1/tests/test_password_policy.py

```diff
@@ -53,6 +55,11 @@ class TestMinimumPasswordAge(TestWithUser):
 
     def test_too_soon(self):
         user = self.get_user()
+        history_item = user.passwordhistoryitem_set.latest()
+        current_locked_until = history_item.locked_until
+        new_locked_until = current_locked_until.replace(year=datetime.MAXYEAR)
+        history_item.locked_until = new_locked_until
+        history_item.save()
         with self.assertRaises(ValidationError):
```

Comment:
It sounds like what we want to test is "When an account is locked, a user cannot change their password", but we're still not entirely testing that, because this is calling `validate_password_age`. What if someone removed that validation on changing a password? The test would still pass. But if we want to simply test the validation function itself, then yes, we should be really clear that's what we're doing. That said, all `validation_password_age` does is call `can_change_password`, which is essentially `return timezone.now() > locked_until`, so writing a unit test for that seems a bit overkill as all it's really testing is that timezone.now and the > operator work as intended. 

We may also want to test, "When a user changes their password, they cannot change it again for another 24 hours." etc.  

I hope I am not being too pedantic :) But I do think it's insanely important for our tests to test what we think they test.